docker/pdo_ccf.dockerfile

```diff
@@ -14,6 +14,8 @@
 # limitations under the License.
 # ------------------------------------------------------------------------------
 
+# syntax = docker/dockerfile:experimental
+
```

Comment:
what is this for?

Reply:
This enables build-kit and the --mount=type extension; will add a comment ...

Reply:
oh right... docker file specifications in comments. i love this tool.